frontend/src/pages/Home.jsx

```diff
@@ -218,6 +219,15 @@ export default function Home() {
       handleCocktailResults();
     }
   };
+
+  const counterDivToggle = () => {
+    if (displayRecipe === false) {
+      buttonDiv.style.display = "none";
+    } else {
+      buttonDiv.style.display = "flex";
+    }
+  };
+
```

Comment:
This cannot be done on re-render ?

Reply:
Not with the way things are done. The states do not empty on dismounting a component but the DOM does, breaking the search function.

Reply:
I'll need to try a few things to be sure but it should be possible.

Reply:
Changed counterDivToggle and replaced it with a state to be more in line with how react works.